.github/workflows/continuous-integration-unit-php.yml

```
@@ -24,12 +24,25 @@ jobs:
     continue-on-error: ${{ matrix.experimental == true }}
     strategy:
       matrix:
-        php: ['7.4', '7.0', '5.6']
+        php: ['7.3', '7.0', '5.6']
```

```suggestion
        php: ['7.0', '5.6']
```

Don't think we really need to test on 7.3. Let's save some time.